spec/other/cli_dispatch_spec.rb

```diff
@@ -14,7 +14,7 @@
   end
 
   it "print a friendly error when ambiguous" do
-    bundle "i"
+    bundle "in"
```

Comment:
Because "bundle i " is now mapped to install

On Fri 5 Jun 2015 at 18:45 Samuel E. Giddins <notifications@github.com>
wrote:

> In spec/other/cli_dispatch_spec.rb
> <https://github.com/bundler/bundler/pull/3722#discussion_r31829523>:
>
> > @@ -14,7 +14,7 @@
> >    end
> >
> >    it "print a friendly error when ambiguous" do
> > -    bundle "i"
> > +    bundle "in"
>
> why is this change necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bundler/bundler/pull/3722/files#r31829523>.
>